### ISSUE TITLE
ImplicitPropertiesAsStrings searchQuery property setting returns no results in certain scenarios.

### ISSUE THREAD

**Issue by MakingThingsWorkToronto** [microsoft-search/pnp-modern-search#804]
**Version used**
Ex: 4.0.0.0 > Develop

**Describe the bug**
The ImplicitPropertiesAsStrings search query property within the SharePointSearchDataSource causes no results to be returned for managed metadata column query (auto generated search managed property).

**To Reproduce**
1) Use an auto generated managed metadata property called owstaxIdXXXX.  
2) Add query template to query directly against the property using KQL owstaxIdXXXX:"Insert Value" 
3) The SharePoint search data source returns no results.  
-- The same query works in v3, within the SearchQueryTool, and in the OOTB search (classic) UI.
-- After some regression testing, it appears the ImplicitPropertiesAsStrings property setting is causing the failure. When this property is removed from the searchQuery property bag, the expected number of SharePoint search results are returned.

**Expected behavior**
Expect the SharePoint search results to be consistent between v3, v4, SearchQueryTool, classic UI etc.. Alternatively need the ability to disable these features.

**Desktop (please complete the following information):**
Windows 10, Google Chrome.

**Additional Information**
Pointers to pnp documentation on how this setting works would be appreciated.

**Comment by wobba** [comment]
@MakingThingsWorkToronto did you test to put your property query in the query template portion?

**Comment by MakingThingsWorkToronto** [comment]
I tried both the QueryTemplate and the static value query text.  Both yielded the same result. Once the ImplicitPropertiesAsStrings was removed from the property bag, it returned as appropriate.  

I'm not sure what the definition of an Implicit Property, but from tech support calls (MS support) they had in the past called the auto generated owstaxIdXXXX columns "implicit properties".  Is it possible that's the issue here? Is there any more documentation on the ImplicitPropertiesAsStrings setting? I can't find any...
